tests/integration/__init__.py

```diff
@@ -554,7 +554,7 @@ def terminate(self):
         self._connectable.clear()
         time.sleep(0.0125)
         terminate_process_pid(self._process.pid)
-        self._process.join()
+#        self._process.join()
```

Comment:
I don't like removing this but at present, doing so produces a stacktrace and prevents the unit tests from running.